CONTRIBUTING.rst

```diff
@@ -188,6 +188,49 @@ To build the docs, run this:
     $ make docs
 
 
+Database migrations
+===================
+
+alembic migrations
+------------------
+
+To create an alembic migration, use your local development environment::
+
+    $ docker-compose run processor bash
+    app@processor:/app$ alembic -c docker/config/alembic.ini revision -m "bug xxx summary here"
+
+Alembic migrations are stored in ``alembic/versions/``. There's a lot of
+material you can crib from there.
+
+To apply alembic migrations, do::
+
+    $ docker-compose run processor bash
+    app@processor:/app$ alembic -c docker/config/alembic.ini upgrade heads
+
+Helper functions in ``socorro.lib.migrations``:
+
+* ``load_stored_proc``
+
+   Loads the latest version of the stored procedures specified. Use this any
+   time you create a new stored procedure or change an existing one.
+
+   Example::
+
+     load_stored_proc(op, ['update_matviews.sql', 'backfill_matviews.sql'])
+
+
+Django migrations
+-----------------
+
+To create a Django migration, user your local development environment::
+
+    $ docker-compose run webapp bash
+    app@webapp:/app$ cd webapp-django
+    app@webapp:/app/webapp-django$ ./manage.py makemigrations
+
+Django migrations are stored in ``webapp-django/crashstats/<appname>/migrations/``.
+
+
```

Comment:
In PR #4512, @lonnen pointed out we had migration docs but they were super wrong. We had some docs in Mana. I moved them here.